.golangci.yml

```diff
@@ -70,6 +70,55 @@ linters-settings:
       # Controller Runtime
       - pkg: sigs.k8s.io/controller-runtime
         alias: ctrl
+      # CABPK
+      - pkg: sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha3
+        alias: bootstrapv1alpha3
+      - pkg: sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha4
+        alias: bootstrapv1alpha4
+      - pkg: sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1beta1
+        alias: bootstrapv1
```

Comment:
should we reserve v1 for real v1 and let the beta imports be consistent e.g bootstrapv1beta1?

Reply:
Hm good question. Up until know we used v1 everywhere (and with everywhere I also mean in a bunch of providers :))

Reply:
I think the main idea behind that is that when we upgrade the version only the import changes and not huge parts of the codebase

Reply:
yes. this is for avoiding churn in the codebase at each release and it kind of makes sense if you read it as the last version in the v1 series (or at least that was the original idea)